system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/DiscoverySystemTest.java

```diff
@@ -25,6 +28,23 @@ public void testNodeDiscoveryRest() throws InterruptedException {
 
         ElasticsearchNodesResponse nodesResponse = new ElasticsearchNodesResponse(tasks, NODE_COUNT);
         assertTrue("Elasticsearch nodes did not discover each other within 5 minutes", nodesResponse.isDiscoverySuccessful());
-    }
+
```

Comment:
Is this testing that the data has actually been created _inside_ the container?
If so, this should be a seperate test, appropriately named. Probably not good to plonk it into the discovery test.